packages/fcl/src/config-utils.js

```diff
@@ -8,3 +8,35 @@ export async function configLens(regex) {
     ])
   )
 }
+
+export async function getStorageConfig() {
+  const storageDefault =
+    STORAGE_OPTIONS[await config.get("fcl.storage.default")]
+  const storageConfig = await config.first([
+    "fcl.storage",
+    "fcl.storage.default",
+  ])
+  STORAGE_OPTIONS[storageConfig] == null &&
+    console.warn(
+      `Invalid fcl.storage value ${storageConfig}. Using default SESSION_STORAGE`
+    )
+  return STORAGE_OPTIONS[storageConfig] || storageDefault
+}
+
+export const STORAGE_OPTIONS = {
+  SESSION_STORAGE: {
```

Comment:
I think I was think one of these object could be passed in as the storage mechanism. In the way you have done it could someone pass in a custom mechanism or does it have to be one of the ones we include?

Reply:
Ah, I see. That would simplify things here.

Reply:
Does this approach need additional validation for user defined storage strategies?

Reply:
dont think so, if it doesnt work, it doesnt work...